packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="oc-ml-s">
+    <div v-if="spaceResource.spaceRoles.manager.length" class="oc-mb-m">
```

Comment:
Fyi: Here is quite some code duplication going on. We like to leave that for now though because the structure of the members object will change as soon as we have the filter. After than we can think about using dedicated components to minimize code duplication.